twilioClient.js

```diff
@@ -7,12 +7,10 @@ module.exports.sendSms = function(to, message) {
     to: to,
     from: config.sendingNumber
 //  mediaUrl: imageUrl
-  }, function(err, data) {
-    if (err) {
-      console.error('Could not notify administrator');
-      console.error(err);
-    } else {
-      console.log('Administrator notified');
-    }
+  }).then(function(data) {
```

Comment:
I think we can put back the `mediaUrl` parameter. That's one of the cool features of the REST API. If you phone number has support for MMS, then it will send it as an embedded image, otherwise it will transform the image in a link that you can check in your browser.

This falls into the category fix broken features.

What do you think?